automation/impossible-to-read-carbonited-metadata-directly-from-within-workflow-check.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+WORKING_DIR=$PWD
+
+JSON_CREDS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN broadinstitute/dsde-toolbox vault read -format=json secret/dsde/firecloud/dev/common/firecloud-account.json | jq '.data'`
+
+BEARER_TOKEN=$(
+        docker \
+            run \
+            --rm \
+            -v "${WORKING_DIR}:/app/populate" \
+            -w /app/populate \
+            broadinstitute/dsp-toolbox \
+            python get_bearer_token.py "dumbledore.admin@test.firecloud.org" "${JSON_CREDS}"
+     )
+
+source ./submission-perf-inc.sh
+
+ENV=alpha
+
+expectedErrorText="does not have storage.objects.get access"
+
+launchSubmission dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace cromwell_carboniting_test_methods read_carbonited_metadata_in_task_body "" "" false false ""
+findLastSubmissionID dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace
+submission1=$submissionID
+
+launchSubmission dumbledore.admin@test.firecloud.org cromwell-tests-billing-project cromwell-tests-workspace cromwell_carboniting_test_methods read_carbonited_metadata_in_workflow_body "" "" false false ""
```

Comment:
is the workflow in a place it could be reviewed?

Reply:
Workflows are listed in the ticket description: https://broadworkbench.atlassian.net/browse/BA-6202
But I adjusted them a bit to make the GCS path an input value. Final versions are uploaded into this workspace: https://firecloud.dsde-alpha.broadinstitute.org/#workspaces/cromwell-tests-billing-project/cromwell-tests-workspace/method-configs